meta-lxatac-software/recipes-support/imx-uuu/imx-uuu_1.5.21.bb

```diff
@@ -0,0 +1,17 @@
+SUMMARY = "Universal Update Utility"
+DESCRIPTION = "Image deployment tool for i.MX chips"
+HOMEPAGE = "https://github.com/NXPmicro/mfgtools"
+
+SRC_URI = "https://github.com/nxp-imx/mfgtools/releases/download/uuu_${PV}/uuu_source-${PV}.tar.gz"
+SRC_URI[sha256sum] = "600be50827b52df4dddf0c7d07da27b103a4576eb445890905c61780e3c36871"
```

Comment:
I don't know if relying on GitHub tarballs is still considered a good idea after [the incident](https://github.blog/2023-02-21-update-on-the-future-stability-of-source-code-archives-and-hashes/).
I don't see a strong commitment to keeping tarball hashes stable in the linked article.
@jluebbe what's your opinion on this?

Reply:
Just fetch via git. This also reduces download size for future releases.

Reply:
In this case, @a3f used the maintainer-uploaded .tar.gz, though. So we could also take this PR as-is.

Reply:
In general, I'd prefer to just fetch via git. This also reduces download size for future releases.

In this case, @a3f used the maintainer-uploaded .tar.gz, though. So we could also take this PR as-is.

Reply:
Ah, using uploaded assets was also a suggestion in the article.
I'll take the PR as-is then.

Reply:
Thanks, I don't mind switching it to git if that's deemed better.

Reply:
@jluebbe Do we need something else, so that users are able to force use of `uuu` when running `labgrid-client bootstrap`?